IMC_R/EDA_Raymond/EDA-R.md

```diff
@@ -0,0 +1,123 @@
+
+![](https://i.imgur.com/KctYyDx.png)
+
+Now, let’s look at our Rotten Tomatoes histogram again.
+
+![](https://i.imgur.com/fCcGusO.png)
+
+This is much better. Although it still isn’t nearly as pretty as the IMDb distribution, we should be able to use this to find the correlation between the two sites. Getting the actual r value for correlation is super easy in R, we just need to use the `cor()` function with two numerical lists. We can also use ggplot, a package from Tidyverse, to create a scatter plot so that we can visualize this relationship. In the `ggplot()` function call, tomatoes is our dataset of movies with non-empty Rotten Tomatoes scores, and we specify IMDb as the x-axis and Rotten Tomatoes as the y-axis. The `geom_point()` component at the end is used to create the actual scatterplot.
+
+![](https://i.imgur.com/6LUfDqw.png)
+
+Our correlation coefficient, 0.377, tells us that there is a moderate relationship between IMDb and Rotten Tomatoes scores, and we can see this in the scatterplot (as IMDb scores increase, Rotten Tomatoes scores tend to increase as well). However, the relationship is a lot weaker than you might’ve expected, which tells us that audiences often perceive movie quality fairly differently from the critics.
+
+## Are Movies on Netflix Above Average?
+Let’s take a look at this one last question to illustrate some other simple EDA techniques you can perform in R. From our last analysis, we’ve seen that IMDb scores are far more normally distributed than Rotten Tomatoes scores. So to answer this question, we’ll see if there’s a statistically significant difference between the average IMDb score of movies on Netflix vs the average score of movies not on Netflix.
+
+First, let’s look at some summary statistics:
+
+![](https://i.imgur.com/KD0aTfJ.png)
+
+Here, it seems like movies on Netflix do indeed have a higher average rating (6.274 compared with 5.807). I also threw in statistics about the year of release as well, to see if movies on Netflix are more recent. And indeed they are.
+
+To get a better understanding of the difference here, we can try to compare these two populations graphically, using side by side boxplots created with ggplot. Now, instead of `geom_point()`, we want to use `geom_boxplot()`, and we want to specify how the groups are assigned, rather than the x-axis.
+
+![](https://i.imgur.com/suSVoYw.png)
+
+Unfortunately, these boxplots still don’t tell us too much. It’s clear again that Netflix does have slightly higher rated movies, but we need to get a measure of statistical significance. This is where a two sample t-test comes in. With the `t.test()` function, we can test the null hypothesis that these two sample populations have the same mean.
```

Comment:
what assumptions does t-test require in order for perform well? how can we trust the results of this analysis?